pkg/controller/traceflow/controller.go

```diff
@@ -410,3 +415,16 @@ func (c *Controller) deallocateTag(name string, tag uint8) {
 		}
 	}
 }
+
+func (c *Controller) validateTraceflow(tf *crdv1alpha1.Traceflow) error {
+	if !tf.Spec.LiveTraffic {
+		srcPod, err := c.podInformer.Lister().Pods(tf.Spec.Source.Namespace).Get(tf.Spec.Source.Pod)
```

Comment:
should we just add a `podLister` member to the `Controller` struct? I think that's what we typically do

Reply:
Sure. Added `podLister` in `Controller` struct.